lib/shoebot.js

```diff
@@ -163,15 +164,17 @@ export default {
         exportErrormsg = null
         exportProcess = spawn(command, {shell: true, detached: true, cwd: scriptDir}, (error, stdout, stderr) => {
           if (error) {
-            console.log(error.stack);
-            console.log('Error code: '+error.code);
-            console.log('Signal received: '+error.signal);
+            console.error(error.stack);
+            console.error('Error code: ' + error.code);
+            console.error('Signal received: ' + error.signal);
           }
```

Comment:
Consider using template literals in these too.

Reply:
good one, fixed some others too